### PR TITLE
Antora Implementation

### PR DESCRIPTION
**Pull request by mulbc**
Lots of work - at best you look into the commit messages or the changes.

```
commit 12ce8bb6dd4ee5b5027b2a661bdf208e766bf470 (HEAD -> antora, origin/antora)
Author: Chris Blum <cblum@redhat.com>
Date:   Wed Sep 23 16:45:07 2020 +0200

    Added Antora rendering

    * Lots of moved files
    * Fixed some ASCIIdoc issues in files
    * Added CODEOWNERS
    * Added nav.adocs
    * Replace MAINTAINER from perf Dockerfile (deprecated)
    * Remove Table of Contents - provided by Antora
    * Remove whitespaces
    * Added titles

commit 10c29237f9889a611573b228458a0473cfd82fa1
Author: Chris Blum <cblum@redhat.com>
Date:   Wed Sep 23 15:28:54 2020 +0200

    Cleaned up repo for Antora move

    * Removed unused YAMLs and pictures
    * Removed docs dir
    * Switched to Antora Github Action
    * Updated Contribution guide and GH Readme
    * Converted Markdown files to ASCIIdoc

commit f084c344a42aa7e438da92e68efc0367b9c74a18
Author: Chris Blum <cblum@redhat.com>
Date:   Wed Sep 23 15:27:19 2020 +0200

    Deleted outdated content

    As discussed in https://docs.google.com/spreadsheets/d/1MnE9oUWB_mkJMSgVaEalIOlL4fUFM66ncmKp9m92ecw/edit?ts=5f65390a#gid=0
```

This is a draft for now, because I want to update the look of the Antora page before we merge - so that we have our official logo in place and don't have the demo drawers. Except for these cosmetic changes, this is considered done from my side, please review

### CODE REVIEW

File: training/modules/ocs4/nav.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* xref:device-replacement.adoc[Replace OCS disks in a running cluster]
+* xref:ocs.adoc[General OCS learning document]
+* xref:ocs4-disconnected-install.adoc[Install OCS in a disconnected environment]
+* xref:ocs4-install-no-ui-1scale.adoc[Scale OCS - 1 node at a time]
+* xref:ocs4-install-no-ui.adoc[Install OCS without the Web UI]
```
Comment (jeanchlopez) [inline training/modules/ocs4/nav.adoc:5]: Most of the titles are too long and do not fit in the left navigation pane on a single line. This does not look rpetty to my eyes. Do we have a way to either make the nav pane wider, shorten the titles or combine both solutions?

File: training/modules/ocs4/pages/ocs.adoc [hunk 2]
```diff
@@ -565,7 +564,7 @@ oc new-project my-database-app
 Now create the new application.
 
 ----
-curl https://raw.githubusercontent.com/red-hat-storage/ocs-training/master/ocp4ocs4/configurable-rails-app.yaml | oc new-app -p STORAGE_CLASS=ocs-storagecluster-ceph-rbd -p VOLUME_CAPACITY=5Gi -f -
+curl https://raw.githubusercontent.com/red-hat-storage/ocs-training/master/training/modules/ocs4/attachments/configurable-rails-app.yaml | oc new-app -p STORAGE_CLASS=ocs-storagecluster-ceph-rbd -p VOLUME_CAPACITY=5Gi -f -
```
Comment (jeanchlopez) [inline training/modules/ocs4/pages/ocs.adoc:567]: Can we add a '-s' after curl so we hide the download extra bits?

File: training/modules/ocs4/nav.adoc (hunk 1 above)
Comment (mulbc) [reply training/modules/ocs4/nav.adoc:5]: Do you have any ideas for shorter titles?
For me it looks like this:
image
so the titles break into a newline, keeping them readable

File: training/modules/ocs4/pages/ocs.adoc (hunk 2 above)
Comment (mulbc) [reply training/modules/ocs4/pages/ocs.adoc]: Implemented by b3fa3ed7c1950cc1a971db87e7c6d129527a42c7

File: training/modules/ocs4/nav.adoc (hunk 1 above)
Comment (jeanchlopez) [reply training/modules/ocs4/nav.adoc:5]: That's exactly what I have too on my end.
1) Deploying on Infra nodes
2) Live disk replacement
3) General content (deploy and use)
4) Disconnected environment install
5) Single node scaling support
6) CLI based install
7) Test deployemnt post-install

Or something like that. We cover OCS in here so no need to put OCS everywhere I think

Comment (jeanchlopez) [reply training/modules/ocs4/nav.adoc:5]: Here is an example I just built
image

Comment (mulbc) [reply training/modules/ocs4/nav.adoc:5]: image
Doesn't fit everywhere... shorting it even more

Comment (mulbc) [reply training/modules/ocs4/nav.adoc]: Implemented by 3eddd1c1e71bd7bcd3bec59cfe41c57538d50b77

**Comment by mulbc** [comment]
Annette requested to re-add `ocs-localdevice-blog.adoc` as an example of how to deploy using local-devices

**Review by jeanchlopez — APPROVED** [review]
I don't see where you have readded ocs-localdevice-blog.adoc. Except that all good so far

**Comment by mulbc** [comment]
Added the design changes - PR is ready for merge now

**Pull request merged**